### ISSUE TITLE
Table of contents problem

### ISSUE THREAD

**Issue by reveurmichael** [open-academy/machine-learning#9]
If we are on chapiter Logistic Regression or before, we cannot see the chapiter Parameter Optimization, which includes Gradient Descent and Loss Function. Instead, Gradient Descent and Loss Function pup out, which is not the desired outcome.

![image](https://user-images.githubusercontent.com/13894593/179940773-2f81146d-685a-4579-9fee-bae5f099e7bc.png)

<img width="230" alt="Snipaste_2022-07-20_16-58-00" src="https://user-images.githubusercontent.com/13894593/179941712-de056422-8642-43e8-afa0-b3bf7457b3ec.png">

**Comment by reveurmichael** [comment]
It's because of the Cache on my PC. Just do a clean rebuild: jupyter-book build --all . , then it's OK.
